### PR TITLE
mgmt deviceregistry, local generate from tsp

### PR DESCRIPTION
**Pull request by XiaofeiCao**
# Description

release request: https://github.com/Azure/sdk-release-request/issues/5107
arc board review: https://github.com/Azure/azure-sdk-pr/issues/1625
specs PR: https://github.com/Azure/azure-rest-api-specs/pull/28780

apiview compared to swagger generated SDK:
https://apiview.dev/Assemblies/Review/90da387f53324bba89095f644611247c/dba2e27ad2044cb1b805a3bb76a26dd9?diffRevisionId=9a421d1e5c67470dbb60a3b4c8f1ae70

Please add an informative description that covers that changes made by the pull request and link all relevant issues.

If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.

# All SDK Contribution checklist:
- [x] **The pull request does not introduce [breaking changes]**
- [x] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [ ] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md).**

## [General Guidelines and Best Practices](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md#developer-guide)
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md#building-and-unit-testing)
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

File: sdk/deviceregistry/azure-resourcemanager-deviceregistry/SAMPLE.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# Code snippets and samples
+
```
Comment (weidongxu-microsoft) [inline sdk/deviceregistry/azure-resourcemanager-deviceregistry/SAMPLE.md:2]: The sample files is generated, but this file is empty?

Comment (XiaofeiCao) [reply sdk/deviceregistry/azure-resourcemanager-deviceregistry/SAMPLE.md:2]: It's due to sample files generated after local repo fix(not sdkIntegration). Let me do a full regen..

File: eng/versioning/version_client.txt [hunk 2]
```diff
@@ -457,6 +457,8 @@ com.azure.resourcemanager:azure-resourcemanager-largeinstance;1.0.0-beta.1;1.0.0
 com.azure.resourcemanager:azure-resourcemanager-migrationdiscoverysap;1.0.0-beta.1;1.0.0-beta.2
 com.azure.resourcemanager:azure-resourcemanager-storageactions;1.0.0-beta.1;1.0.0-beta.2
 com.azure.resourcemanager:azure-resourcemanager-standbypool;1.0.0-beta.1;1.0.0-beta.1
+com.azure.resourcemanager:azure-resourcemanager-deviceregistry;1.0.0-beta.1;1.0.0-beta.1
+com.azure.resourcemanager:None;1.0.0-beta.1;1.0.0-beta.1
```
Comment (weidongxu-microsoft) [inline eng/versioning/version_client.txt:461]: delete this

**Review by weidongxu-microsoft — APPROVED** [review]

**Comment by weidongxu-microsoft** [comment]
you probably had to use a good commit ID, and ignore the codegen verify step
https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/vision/ci.yml#L48

**Comment by XiaofeiCao** [comment]
> you probably had to use a good commit ID
> 
> and maybe you would have to ignore the codegen verify step https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/vision/ci.yml#L48

Thanks, guess I'll skip it. service's example files have issues:
https://github.com/Azure/azure-rest-api-specs/pull/28783/files

**Comment by weidongxu-microsoft** [comment]
Try it first.

The verification should already skip non-Java file, as well as tests and samples folder.
https://github.com/Azure/azure-sdk-for-java/blob/main/eng/scripts/TypeSpec-Compare-CurrentToCodegeneration.ps1#L73

**Pull request merged**